src/app/system/manage-scheduler-jobs/view-history-scheduler-job/view-history-scheduler-job.component.html

```diff
@@ -0,0 +1,49 @@
+<div class="container">
+
+  <div fxLayout="row" fxLayoutGap="20px">
+
+    <mat-form-field fxFlex>
+      <mat-label>Filter By Version</mat-label>
+      <input matInput (keyup)="applyFilter($event.target.value)">
+    </mat-form-field>
+  </div>
+
+  <div class="mat-elevation-z8">
+
+    <table mat-table [dataSource]="dataSource" matSort>
+
+      <ng-container matColumnDef="version">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Version </th>
+        <td mat-cell *matCellDef="let jobHistoryData"> {{ jobHistoryData.version }} </td>
+      </ng-container>
+
+      <ng-container matColumnDef="run_start_time">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Run Start Time </th>
+        <td mat-cell *matCellDef="let jobHistoryData"> {{ jobHistoryData.jobRunStartTime }} </td>
+      </ng-container>
+
+      <ng-container matColumnDef="status">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Status </th>
+        <td mat-cell *matCellDef="let jobHistoryData"> {{ jobHistoryData.status }} </td>
+      </ng-container>
+
+      <ng-container matColumnDef="run_type">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Run Type </th>
+        <td mat-cell *matCellDef="let jobHistoryData">{{ jobHistoryData.triggerType }}</td>
+      </ng-container>
+
+      <ng-container matColumnDef="error_log">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Error Log </th>
+        <td mat-cell *matCellDef="let jobHistoryData"> </td>
```

Comment:
@BLasan Can you please make these changes?

The errorlog status needs to be displayed as in the community app which also displays a log modal on click.
![Screenshot from 2020-05-17 11-47-54](https://user-images.githubusercontent.com/36980003/82137278-49142600-9834-11ea-8987-4482e97a7f80.png)

Screenshot of  error log dailog box:
![Screenshot from 2020-05-17 11-50-34](https://user-images.githubusercontent.com/36980003/82137318-9395a280-9834-11ea-9655-c0653719fe49.png)

Reply:
I'll refactor them